src/mlpipeline/mlflow.py

```diff
@@ -46,12 +47,17 @@ def tracking_uri(self, uri):
         if not isinstance(uri, str):
             raise TypeError("Input needs to be a String")
 
+        separator = "\\"
+
+        if "/" in uri:
+            separator = '/'
+
```

Comment:
If I remove the func created in experiment.py 
  # @property
    # def abs_mlruns_path(self):
    #     path = os.path.abspath(os.path.dirname(""))
    #     mlflow_uri_path = path.split("nlp-web-scrapping")[0]
    #     separator = "\\"
    #     if "/" in mlflow_uri_path:
    #         separator = '/'
    #     mlflow_uri_path = mlflow_uri_path + "nlp-web-scrapping" + separator + "mlruns"
    #     return mlflow_uri_path

and I chage:
inside  experiment.py, function save_to_mlflow()  line 106:
when Instantiating a new MlFlow object doing
mlflow = MlFlow(self.name, self.path)

I set the path 
path = os.path.abspath(os.path.dirname(""))

Question can you change those things and amend the code to make sure this is fully working please? 
The code in this branch is still using that function.
meaning what you have added is not adding any extra funcionality. "duplicated code"

Reply:
I'm a little confused by this one

Reply:
Nat I'll try to be more clear. 

Before you created this branch, I already added a funcion to address the foward/backward slide in mlflow uri path.
Do you remember I told you about this funcion in Teams?

However, you also addressed the issue in this pull request but in a different part of the code.
This means, you created a branch with my funcion already in place to fix the problem.
And you added your solution on top of that. Meaning we where doing same thing twice in different places. 

I'm happy to keep your code. 
That's why I commented out the code I wrote prior to your changes in the commit name: 
Reverting my changes to check Nats code

Since now this branch does only have your lines to fix the foward or backwards slice. 
Could you double check that your code changes work?

Thanks